### PR TITLE
docs: FPS Meter

### PR DESCRIPTION
**Pull request by Ombuweb**

### CODE REVIEW

File: content/guide/core/fps-meter.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+## FPS Meter API
+
+### addCallback
+
+```ts
+const callbackId = addCallback(callback: (fps: number, minFps?: number) => void): number
+```
+
+Adds a callback function to be called each time FPS data is ready. The a unique `number` id which can be used to remove this callback later.
```
Comment (ashutosh887) [inline content/guide/core/fps-meter.md:59]: Implement a callback function that will be executed whenever FPS data becomes available. Assign a unique numerical identifier (`number`) to this callback, facilitating its future removal, if required.

Comment (ashutosh887) [inline content/guide/core/fps-meter.md]: Implement a callback function that will be executed whenever FPS data becomes available. Assign a unique numerical identifier (`number`) to this callback, facilitating its future removal, if required.

**Review by ashutosh887 — CHANGES_REQUESTED** [review]

**Comment by ashutosh887** [comment]
@NathanWalker I've requested some changes!

File: content/guide/core/fps-meter.md (hunk 1 above)
Comment (NathanWalker) [inline content/guide/core/fps-meter.md]:
```suggestion
Implement a callback function that will be executed whenever FPS data becomes available. Assign a unique identifier (`number`) to this callback, facilitating its future removal, if needed.
```

File: content/guide/core/fps-meter.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+---
+title: FPS Meter
+---
+
+This module allows you capture the frames-per-second metrics of your application.
```
Comment (NathanWalker) [inline content/guide/core/fps-meter.md]:
```suggestion
FPS Meter allows you capture the frames-per-second metrics of your application.
```

File: content/guide/core/fps-meter.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+---
+title: FPS Meter
+---
+
+FPS Meter allows you capture the frames-per-second metrics of your application.
+
+## Using FPS Meter
+
+### Measuring the fps
```
Comment (NathanWalker) [inline content/guide/core/fps-meter.md]:
```suggestion
### Measuring FPS
```

File: content/guide/core/fps-meter.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+title: FPS Meter
+---
+
+FPS Meter allows you capture the frames-per-second metrics of your application.
+
+## Using FPS Meter
+
+### Measuring FPS
+
+To use the module, first register a callback that will receive the FPS metrics using the [addCallBack](#addcallback) method. Then, call the [start](#start) method to start measuring the fps.
```
Comment (NathanWalker) [inline content/guide/core/fps-meter.md]:
```suggestion
Register a callback that will receive the FPS metrics using the [addCallBack](#addcallback) method. Then call the [start](#start) method to begin measuring.
```

File: content/guide/core/fps-meter.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+    console.log(minFps?.toFixed(2))
+  })
+  
+  start()
+  console.log('Is running: ', running())
+}
+```
+
+### Stop measuring fps
```
Comment (NathanWalker) [inline content/guide/core/fps-meter.md]:
```suggestion
### Stop measuring FPS
```

File: content/guide/core/fps-meter.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  })
+  
+  start()
+  console.log('Is running: ', running())
+}
+```
+
+### Stop measuring FPS
+
+To stop measuring the fps, remove the registered callback using its id and then call the module's [stop](#stop) method.
```
Comment (NathanWalker) [inline content/guide/core/fps-meter.md]:
```suggestion
Remove the registered callback using its id and then call the [stop](#stop) method.
```

**Pull request merged**